source/RuuviTagApp.mc

```diff
@@ -27,6 +28,39 @@ class RuuviTagApp extends Application.AppBase {
     bleDelegate_ = new RuuviTagBleDelegate(method(:onRuuviTagData));
   }
 
+  private function verifySensorProperty (sensorMacAddress, sensorAlias) {
+    var sensorMacAddressCA = sensorMacAddress.toCharArray();
+
+    // verify if format of user-configurable hardware address is reasonable
+    if ((sensorMacAddress.length() == 14) && 
+        (sensorMacAddressCA[2] ==
+         sensorMacAddressCA[5] ==
+         sensorMacAddressCA[8] ==
+         sensorMacAddressCA[11] == ':')) {
+        return true;
+      }
+    return false;
+  }
+
+  function onStart(state) {
+    for (var i = 1; i < 6; i++) {
+      var sensor = "sensor-" + i.format("%02d");
+      var sensorMacAddress;
+      var sensorAlias;
+
+      try {
+        sensorMacAddress = Application.Properties.getValue(sensor + "-mac-address");
+        sensorAlias = Application.Properties.getValue(sensor + "-alias");
+
+        if (verifySensorProperty(sensorMacAddress, sensorAlias)) {
+          sensorAliases_[sensorMacAddress.toUpper()] = sensorAlias;
+        }
+      }
+      catch(ex) {
+      }

```

Comment:
Within this catch block it would be useful to describe (as a code comment) when/why it's expected that the code above will throw an exception.